packages/coordinator/tests/services/translation/response/release/release-response.spec.ts

```diff
@@ -251,6 +252,11 @@ describe("bundle resources", () => {
     const organizations = getOrganizations(result)
     expect(organizations).toHaveLength(1)
   })
+  
+  test("does not contain HealthcareService resource", () => {
```

Comment:
I would prefer if we just updated our models to never expect healthcare service rather than leaving it as an optional field and testing that we don't populate it.

Reply:
Yeah I agree that makes more sense, I'll change that to never be expected